docs/README-AWS.md

```diff
@@ -7,10 +7,11 @@ The Elastic Serverless Forwarder is an AWS Lambda function that ships logs from
 - Direct SQS message payload input
 - S3 SQS Event Notifications input
 - Kinesis Data Stream input
+- Cloudwatch logs events input
```

Comment:
I like "CloudWatch logs subscription filter input". I know its a little wordy but I think we can be a little wordy in the doc to make it more easier for the users to understand.